.github/workflows/daily.yml

```diff
@@ -32,12 +32,60 @@ jobs:
         # 1. RUSTSEC-2021-0020 - Not impacted, see #7723
         # 2. RUSTSEC-2020-0146 - Not impacted, see #7826
         run: |
-          cargo audit --ignore RUSTSEC-2021-0020 --ignore RUSTSEC-2020-0146 >> $MESSAGE_PAYLOAD_FILE
-      - uses: ./.github/actions/slack-file
-        with:
-          webhook: ${{ secrets.WEBHOOK_AUDIT }}
-          payload-file: ${{ env.MESSAGE_PAYLOAD_FILE }}
+          cargo audit --ignore RUSTSEC-2021-0020 --ignore RUSTSEC-2020-0146 > $AUDIT_SUMMARY_FILE
+      - name: create issue on failure
+        uses: actions/github-script@v3
```

Comment:
what would it take to turn this in to an action specific to creating/updating issues?   It would be generically useful, and wouldn't result in a lot of copy/paste.

Reply:
sounds reasonable. this is the first use case i came across.

Reply:
no dice, composite "uses" is not supported atm (https://stackoverflow.com/questions/64055230/nested-templates-calling-a-yaml-file-from-another-yaml-file-in-github-actions)

Reply:
looks like this should work https://github.com/actions/github-script#run-a-separate-file-with-an-async-function